### PR TITLE
[WIP] add blog post for mongolite

### PR DESCRIPTION
**Pull request by jeroenooms**

### CODE REVIEW

File: _posts/2017-03-08-mongolite.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+# Make a connection
+con <- mongolite::mongo("diamonds", 
+  url = "mongodb://readwrite:test@ds043942.mongolab.com:43942/jeroen_test")
+
+# Find diamonds with: cut == Premium & price < 500
+mydata <- con$find('{"cut" : "Premium", "price" : { "$lt" : 500 } }')
+print(mydata)
+```
+
+In my experience, a simple interface is critical to get started. Obviously more advanced interfaces are available as well, but this will gets you up to speed right way if you just need the data and get on with your job.
```
Comment (sckott) [inline _posts/2017-03-08-mongolite.md]: @jeroenooms i assume in _more advanced interfaces are available as well_ you mean in `mongolite`, yes? maybe say that if so

File: _posts/2017-03-08-mongolite.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ - `insert()` now substitutes dots in key (column) names with underscores
+ - Various fixes in `update()`, support for upsert
+ - Add unit tests from bson spec (some tests fail in mongo-c-driver)
+ - Switch to new C driver API mongoc_collection_find_with_opts()
+ - Add R_registerRoutines() call to please CMD check
+
+
+## Tell us what you think
+
+At ropensci we're interested to hear who is using R and how. If you decide to give mongolite a try, please share your experiences, problems and suggestions. The best way to get in touch is join the ropensci slack channel.
```
Comment (sckott) [inline _posts/2017-03-08-mongolite.md]: s/`ropensci`/`rOpensci`

File: _posts/2017-03-08-mongolite.md (hunk 1 above)
Comment (stefaniebutland) [inline _posts/2017-03-08-mongolite.md]: in same sentence should be "get you up to speed"
s/gets/get

File: _posts/2017-03-08-mongolite.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+We love to hear from academic users, but also industry applications of R and syngery between the industry and open source scientific software.
```
Comment (stefaniebutland) [inline _posts/2017-03-08-mongolite.md]: s/syngery/synergy

File: _posts/2017-03-08-mongolite.md (hunk 2 above)
Comment (stefaniebutland) [inline _posts/2017-03-08-mongolite.md]: actually: s/ropensci/rOpenSci
at least twice in post

**Comment by stefaniebutland** [comment]
Seems like mongolite and your bookdown is a mighty big deal. Expecting many fan-tweets.
I'll get you an answer re: best way to get in touch.

File: _posts/2017-03-08-mongolite.md (hunk 2 above)
Comment (sckott) [inline _posts/2017-03-08-mongolite.md]: woops, yeah, meant `rOpenSci`

**Comment by jeroenooms** [comment]
Thanks! I have edited the post, bumped the date and squashed the commits. I think it's ready now.

I put the date on 03/10, but perhaps we can post it to day? That way it gets displayed at the top of all aggregators as newest content.

File: _posts/2017-03-10-mongolite.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+ - Add unit tests from bson spec (some tests fail in mongo-c-driver)
+ - Switch to new C driver API mongoc_collection_find_with_opts()
+ - Add R_registerRoutines() call to please CMD check
+
+
+## Tell us what you think
+
+At rOpenSci we're interested to hear who is using R and how. If you decide to give mongolite a try, please share your experiences and suggestions. Open an [issue on github](https://github.com/jeroenooms/mongolite/issues) for specific problems and feature requests, or join the rOpenSci [slack](https://ropensci.slack.com) to talk about mongolite or other rOpenSci packages. Or just come say hi and hang out :)
+
+We both love hearing from academic users, but also industry applications of R and synergy between the industry and open source scientific software.
```
Comment (stefaniebutland) [inline _posts/2017-03-10-mongolite.md]: replacement for last sentence (grammar fix):
"We love hearing from both academic and industry users, including industry applications of R and synergy between the industry and open source scientific software."

Comment (stefaniebutland) [inline _posts/2017-03-10-mongolite.md]: Looks good to go as you suggested @jeroenooms

**Pull request merged**